backend/uclapi/.env.example

```diff
@@ -22,6 +22,13 @@ UCLAPI_DOMAIN=uclapi.com
 # See https://dryan.com/articles/elb-django-allowed-hosts/ for more info.
 UCLAPI_RUNNING_ON_AWS_ELB=False
 
+# Set URLs that may not be used as callback URLs (for security reasons).
+UCLAPI_OAUTH_CALLBACK_DENIED_URLS=localhost;127.0.0.1
```

Comment:
Yes, it should block all ports at those domains. The production and staging blocks will also include staging.ninja and uclapi.com to prevent anybody trying to make our code redirect round in a loop.